microsite/docusaurus.config.ts

```diff
@@ -444,5 +429,8 @@ const config: Config = {
       ],
     },
   } satisfies Preset.ThemeConfig,
+  future: {
+    experimental_faster: true,
```

Comment:
I "think" if you avoid this change we could still move forward with the version bump.

Reply:
👋 Can we update the description with a bit more context here? I would assume that we can run the docsite on a different version of React from the main repo and it wouldn't need to tie into your work in https://github.com/backstage/backstage/pull/28015.